### PR TITLE
Added documentation on cluster suspension

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement

New docs on cluster suspension

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/suspend-cluster.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+.. contents::
+   :local:
+
+
+.. _what-is-suspendion:
+
+What is cluster suspension
+==========================
+
+Cluster suspension is a feature that enables you to pause the computation on
```
Comment (SStorm) [inline docs/suspend-cluster.rst:22]: Is "computation" the right word here? Seems unusual :)

Maybe something like "Cluster suspension is a feature that allows you to temporarily pause your cluster, retaining any data you have on it."

File: docs/suspend-cluster.rst [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+your cluster temporarily. An example situation might be that the project
+you're working on has been put on hold. In that case, you don't need the computation, but you still need the storage to preserve your data. The cost of running a cluster is split into two parts: **Compute** and **Storage**. The benefit here is that while the cluster is suspended, you are only charged for the storage.
+
+
+.. _how-to-suspend-a-cluster:
+
+Suspend a cluster
+=================
+
+To scale a cluster, you need to know what project the cluster belongs to. Go to
```
Comment (SStorm) [inline docs/suspend-cluster.rst:32]:
```suggestion
To suspend a cluster, you need to know what project the cluster belongs to. Go to
```

File: docs/suspend-cluster.rst [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+- To suspend a cluster, you must be a Project Admin
+
+- You can suspend and resume clusters at any time
+
+- While your cluster is suspended, you will still be billed for the storage and
+  the billing cycle is unaffected
+
+- Cluster backups do not expire while a cluster is suspended
+
+- Admin UI of a cluster is unavailable while the cluster is suspended
```
Comment (SStorm) [inline docs/suspend-cluster.rst:90]:
```suggestion
- The Admin UI of a cluster is unavailable while the cluster is suspended
```

File: docs/suspend-cluster.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+
+.. _what-is-suspendion:
+
+What is cluster suspension
+==========================
+
+Cluster suspension is a feature that enables you to pause the computation on
+your cluster temporarily. An example situation might be that the project
+you're working on has been put on hold. In that case, you don't need the computation, but you still need the storage to preserve your data. The cost of running a cluster is split into two parts: **Compute** and **Storage**. The benefit here is that while the cluster is suspended, you are only charged for the storage.
```
Comment (SStorm) [inline docs/suspend-cluster.rst:24]: Maybe remove the sentence "in that case ..." as it seems a bit redundant.

Comment (SStorm) [reply docs/suspend-cluster.rst:24]: The rest of the document seems line-wrapped, while these two lines are pretty long - is there some convention here?

Comment (matkuliak) [reply docs/suspend-cluster.rst:24]: Ah, my auto-wrapper keeps acting up. Yeah, the convention is line-wrap to 79 chars.

File: docs/suspend-cluster.rst [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        :alt: Cloud Console cluster suspension notification
+
+
+.. _how-to-resume-a-cluster:
+
+Resume a cluster
+================
+
+To resume your cluster, simply press the "Go to *Preferences* to resume
+cluster" link in the top bar, or navigate to the Cluster preferences page
```
Comment (matthijskrul) [inline docs/suspend-cluster.rst:64]: Preferences

File: docs/suspend-cluster.rst [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+After pressing the *Resume cluster* button in the top right, your cluster will
+be resumed.
+
+.. image:: _assets/img/cluster-resume.png
+   :alt: Cloud Console resume cluster
+
+
+.. _additional-info:
+
+Additional info on cluster suspension
```
Comment (matthijskrul) [inline docs/suspend-cluster.rst:76]: Can we give this a more meaningful title maybe?

**Review by matthijskrul — APPROVED** [review]
Just a few nitpicks. Looks good! Keep in mind any change in the Console also requires updating the Console overview in the Reference :)

**Review by SStorm — APPROVED** [review]

**Pull request merged**